examples/src/example_gcd.cpp

```diff
@@ -0,0 +1,13 @@
+#include <dbg_macro/dbg.h>
```

Comment:
the additional two examples are nice, but to be honest, I don't think they add anything in value (documentation-wise and test-coverage-wise) over the existing example.

Reply:
You are correct. I will remove `gcd.cpp`.